src/components/layout/footer.js

```diff
@@ -546,30 +519,64 @@ const Footer = () => {
-                                components={[
-                                    <BoldLink key={0} target="_blank" to="/responsible-trading/" />,
-                                ]}
-                            />
-                        </StyledText>
-                    </Row>
-                </StyledContainer>
-            </Disclaimer>
-            <Copy />
+                        </DisclaimerParagraph>
+                        <RiskWarning>
+                            <DisclaimerParagraph no_margin="true">
+                                <Localize
+                                    translate_text="<1>RISK WARNING:</1> The financial products offered via this website include digitals, contracts for difference (CFDs), and other complex derivatives and financial products. Trading options may not be suitable for everyone. Trading CFDs carries a high level of risk since leverage can work both to your advantage and disadvantage. As a result, the products offered on this website may not be suitable for all investors because of the risk of losing all of your invested capital. You should never invest money that you cannot afford to lose, and never trade with borrowed money. Before trading in the complex financial products offered, please be sure to understand the risks involved and learn about <0>Secure and responsible trading.</0>"
+                                    components={[
+                                        <BoldLink
+                                            key={0}
+                                            target="_blank"
+                                            to="/responsible-trading/"
+                                        />,
+                                        <strong key={1} />,
+                                    ]}
+                                />
+                            </DisclaimerParagraph>
+                        </RiskWarning>
+                    </Disclaimer>
+                    <Copyright>
+                        <CopyrightIc width="16px" />
+                        <Text ml="0.4rem">{localize('2020 Deriv | All rights reserved')}</Text>
+                    </Copyright>
+                    <SocialWrapper>
+                        <ExternalLink
+                            href="https://www.facebook.com/derivdotcom/"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            <Facebook />
+                        </ExternalLink>
+                        <ExternalLink
+                            href="https://twitter.com/derivdotcom"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            <Twitter />
+                        </ExternalLink>
+                        <ExternalLink
+                            href="https://www.instagram.com/derivdotcom/"
```

Comment:
```suggestion
                            href="https://www.instagram.com/deriv_official/"
```

they change the name of instagram link